.github/workflows/trigger-webui.yml

```diff
@@ -32,6 +32,7 @@ jobs:
     runs-on: ubuntu-24.04
     # the default workflow token cannot read our org membership, for deciding who is allowed to trigger tests
     environment: gh-cockpituous
+    container: registry.fedoraproject.org/fedora:40
```

Comment:
Why Fedora 40, it is already EOF.

Reply:
It's just a revert of the previous commit, as mentioned in the commit message. 
I can update and remove the mention this is revert.